### PR TITLE
Adds minor brain damage to holywater

### PR DESCRIPTION
**Pull request by Regen1**
Because it's made from mercury, and to stop people from just creating
"validhunt smoke"
Adds a notice when you take brain damage from holywater as well.

### CODE REVIEW

**Comment by Alex6511** [comment]
While I support this, it might be a good idea to add some sort of feedback to the player that drinking fuckloads of holy water is harming them, as it's not explicitly clear.

**Comment by Spacemanspark** [comment]
This will help stop validgaming a bit.

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -471,6 +471,9 @@ datum
 			process_flags = ORGANIC | SYNTHETIC
 
 			on_mob_life(var/mob/living/M as mob)
+				if(prob(70))
+					M.adjustBrainLoss(1)
+					M << "<span class = 'danger'>You feel your belief in religion bolstering</span>"
```
Comment (CorporationStation) [inline code/modules/reagents/Chemistry-Reagents.dm]: *Extremely* picky, but there's no period/exclamation mark here.

**Comment by Alex6511** [comment]
I noticed you add the notification, thanks. I think that will really help.

**Comment by Fox-McCloud** [comment]
:-1: 

It's holy water that's pure--it has positive effects for every single other avenue/aspects of how it functions, from cleaning floors, to healing plants/functioning better for fishtanks. Likewise, it just means people will mix in mannitol to their smoke mixes to counter-act the effects.

**Comment by Regen1** [comment]
It's not holy water as in "ahem holy water"
It is literally made of mercury, water and wine.
What this change does it make give security a reason NOT to use holy water grenades, and because it is potentially dangerous, it gives a reason for security to actually arrest people using it, because it gives brain damage.
We could also couple it with adding to the SOP that using this as a smoke in large crowds is not allowed because of the brain damage it causes.

It is a better fix than what we currently have, cause holy water is being heavily abused

**Comment by Fox-McCloud** [comment]
Again, holy water is literally the only way to dealing with a cult; there's no other way to deconvert them--whether there's a recipe or not isn't going to change that (it's worth noting that the recipe didn't exist until recently), as the Chaplain can generate hundreds of thousands of units in mere seconds--likewise, anyone with a Bible (one freely available and you can print them) can do similar on a lesser scale.

This will only induce players to use holy water in conjunction with mannitol, thus negating the effects of it. If you're going to suggest making it against SoP, then it doesn't matter if it has brain damage or not (I'd also suggest that security won't give a crap about people utilizing holy water grenades when its confirmed that there is a cult on the loose).

**Comment by Spacemanspark** [comment]
Security, unfortunately, gives absolutely zero shits in regards to SoP when it comes to validhunting.

**Comment by Robertha89** [comment]
The solution is to make holy water from Water, Wine and nutriment instead.

**Comment by Regen1** [comment]
Why would that fix anything Robertha? This is to fix Validhunting, not giving a reason for the mercury

**Comment by Regen1** [comment]
@Fox-McCloud This won't stop people from deconverting cult, it does very minor brain damage, but what it does do is stop chemists from creating huge smoke clouds of validhunt101.
Sure you can counter it, but at least it adds a little more effort.

**Comment by Robertha89** [comment]
the way to fix valid hunting is making the chances of deconvertion RNG instead of OH, deconvertedlawl, not adding lolrandum damage. 

Also, only the person with mindrole Chaplin should be able to make holy water by hitting water containers with a bible, otherwise it is basically an bug.

**Comment by tigercat2000** [comment]
@Robertha89 ... that's not 'basically a bug' it's fully intended

**Comment by Fox-McCloud** [comment]
It...won't discourage them though; mannitol literally takes an additional 2 seconds (literally) to make.

**Comment by Fox-McCloud** [comment]
update, see this:

**Pull request closed without merging**